Modules/ossaudiodev.c

```diff
@@ -1043,7 +1043,7 @@ static PyMethodDef ossaudiodev_methods[] = {
 
 
 #define _EXPORT_INT(mod, name) \
-  if (PyModule_AddIntConstant(mod, #name, (long) (name)) == -1) return NULL;
+  if (PyModule_AddIntConstant(mod, #name, (long) (name)) == -1) {Py_DECREF(mod); return NULL;}
```

Comment:
It would prefer to rewrite the function as an "_module_exec" function and do the DECREF in the PyInit function.